src/main/java/com/github/messenger4j/send/recipient/UserRefRecipient.java

```diff
@@ -0,0 +1,28 @@
+package com.github.messenger4j.send.recipient;
+
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.ToString;
+
+/**
+ * @author Joe Tindale
+ * @since 1.0.0
```

Comment:
@maxgrabenhorst I guess we will need to update the @since tag to match whichever version this is released in.